itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationScenariosAcrossInstances.java

```diff
@@ -1504,12 +1504,20 @@ public void testDumpExternalTableSetFalse() throws Throwable {
             .run("use " + replicatedDbName)
             .run("show tables like 't3'")
             .verifyResult("t3")
+            .verifyExternalTable(replicatedDbName, "t3",
```

Comment:
In this logic, the final table location is expected to be same as primary warehouse location. How do we verify if it is re-allocated by replica warehouse?
Try to drop the table and delete the location before triggering replication to see if replica warehouse create the same path as source and copy right files.

Reply:
For that we need support to copy the external table data from source to destination which will be part of another JIRA. Hence right now this test only tests that the location on the source and target replica is same and since they happen to be on the same file system in this test, the data will also be available.